src/views/open-api-docs-preview/components/open-api-preview-inputs/open-api-preview-inputs.module.css

```diff
@@ -0,0 +1,55 @@
+.root {
+	--highlight-color: var(--token-color-foreground-highlight);
+	--border-width: 4px;
+
+	background: black;
+	border-left: var(--border-width) solid var(--highlight-color);
+	bottom: 0;
+	position: fixed;
+	right: var(--border-width);
+	top: 0;
+	transition: transform 0.3s;
+	width: min(80vw, 800px);
+
+	&.isCollapsed {
+		transform: translateX(100%);
+	}
+}
+
+.scrollableContent {
+	overflow: auto;
+	padding: 1rem;
+	max-height: 100%;
+}
+
+.inputs {
+	display: flex;
+	flex-direction: column;
+	gap: 16px;
+}
+
+.collapseButtonLayout {
+	position: absolute;
+	top: 50%;
+	right: 100%;
+	transform: translateY(-50%);
+}
+
+.collapseButton {
+	padding: 2rem;
+	border-top-left-radius: 50%;
+	border-bottom-left-radius: 50%;
+	border: none;
+	background: var(--highlight-color);
+	color: black;
+	cursor: pointer;
+}
+
+.generateButton {
+	flex-shrink: 0;
+
+	/* Swap blue and purple to match the --highlight-color */
```

Comment:
Did this as a bit of a hack, feels worth it to have the preview UI stand out a bit. Will definitely be removing this once we have the HDS components needed to implement the ✨ [actual designs](https://www.figma.com/file/wjoC3ZVsA1kDuPsFLjEFuY/API-Template?type=design&node-id=720-3306&mode=design&t=fmzKCVL2rUMWKEpb-4) ✨ .